### DISCUSSION TITLE
Using atoms in Global Styles

### DISCUSSION THREAD

**Discussion by douglaseggleton** [vanilla-extract-css/vanilla-extract#109] [Q&A] [answered]
Is it possible to use atoms in global styles? :-)

```tsx
import { style, globalStyle } from "@vanilla-extract/css";
import { atoms } from "../../atoms.css";

// atoms({
//   background: 'canvas',
// })

globalStyle(`body`, {
  background: 'canvas', // How can I use the atom here?
});
```

**Comment by mattcompiles** [comment, marked as answer]
The `atoms` function produced by Sprinkles is an abstraction on top of the `style` function from `@vanilla-extract/css`. You can't modify the behaviour of the `style` or `globalStyle` function.

**Reply by douglaseggleton** [reply]
Ah got it! Thanks - I misunderstood :-)
